sys/linux/socket_netlink_generic_ethtool.txt

```diff
@@ -12,11 +12,30 @@ type msghdr_ethtool[CMD, POLICY] msghdr_netlink[netlink_msg_t[genl_ethtool_famil
 syz_genetlink_get_family_id$ethtool(name ptr[in, string["ethtool"]]) genl_ethtool_family_id
 
 sendmsg$ETHTOOL_MSG_STRSET_GET(fd sock_nl_generic, msg ptr[in, msghdr_ethtool[ETHTOOL_MSG_STRSET_GET, strset_get_policy]], f flags[send_flags])
```

Comment:
Please reorder these to either match the order in `include/uapi/linux/ethtool_netlink.h` or first all getters and then all setters.

Reply:
[fixed](https://github.com/google/syzkaller/pull/1903/files#diff-4277f1131ebd61c5de5aff5107a42d09L15-L19)